src/app.ts

```diff
@@ -1,6 +1,72 @@
+type ResultData = {
+    id: string,
```

Comment:
Codacy has a fix for the issue: [Replace `··id:·string,` with `id:·string;`](https://app.codacy.com/gh/dekunleroy/pipeline_assessment/pullRequest?prid=9962768)
```suggestion
  id: string;
```